pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -630,6 +630,8 @@ let
         };
       };
 
+      contextmapper.context-mapper-vscode-extension = callPackage ./context-mapper-vscode-extension { };
```

```suggestion
      contextmapper.context-mapper-vscode-extension = callPackage ./contextmapper.context-mapper-vscode-extension { };
```
Directory should equate extension name. Which per upstream, includes the publisher name. (Or else it would clash names.)